molgenis-app/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.1"
+services:
+    molgenis:
+      image: registry.molgenis.org/molgenis/molgenis-app:${TAG:-latest}
+      ports:
+        - "8081:8080"
+      environment:
+        - molgenis.home=/home/molgenis
+        - opencpu.uri.host=opencpu
+        - elasticsearch.transport.addresses=elasticsearch:9300
+        - db_uri=jdbc:postgresql://postgres/molgenis
+        - db_user=molgenis
+        - db_password=molgenis
+        - admin.password=admin
+        - "CATALINA_OPTS=-Xmx1g"
+
+    postgres:
+      image: postgres:9.6-alpine
+      environment:
+          - POSTGRES_USER=molgenis
+          - POSTGRES_PASSWORD=molgenis
+          - POSTGRES_DB=molgenis
+
+    elasticsearch:
+        image: docker.elastic.co/elasticsearch/elasticsearch:5.5.1
+        environment:
+          - cluster.name=molgenis
+          - bootstrap.memory_lock=true
+          - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
```

Comment:
http://molgenis.org/wiki/GccSopCreateMolgenisVM#InstallElasticsearch states ```sed -i -- 's/-Xmx2g/-Xmx1g/g' /etc/elasticsearch/jvm.options```

Reply:
ES_JAVA_OPTS are appended to the command line and therefore override the defaults listed in jvm.options.